appserver/admingui/common/src/main/java/org/glassfish/admingui/common/util/RestUtil.java

```diff
@@ -997,6 +1002,39 @@ public boolean verify(String host, SSLSession sslSession) {
         }
     }
 
+    public static synchronized InputStream getInputStream(String host, int port, String urlStr, String password, boolean isSecure) throws IOException {
```

Comment:
This change is unnecessary:

1. You are changing server side code instead of fixing the client code (BaseSeleniumTestClass) The client should use "http" or " https" URL depending on what it needs to connect to.
2. captureLog() is not needed for the Jenkins job since the script itself backs up the server log. We are unnecessarily copying the log into 2 places.

Reply:
Especially, this function did not work even if I pass especially https. I tried. 

If we send http and secure admin is on, it automatically redirecting to login page(https). 

But, we need captureLog when we try run them locally or individually. Hence, fixed this method instead of removing.